js/ui/marker.js

```diff
@@ -13,6 +13,10 @@ var Point = require('point-geometry');
  * @param {HTMLElement=} element DOM element to use as a marker (creates a div element by default)
  * @param {Object=} options
  * @param {PointLike=} options.offset The offset in pixels as a [`PointLike`](#PointLike) object to apply relative to the element's top left corner. Negatives indicate left and up.
+ * @param {boolean=} [options.pitchScale=false] Whether the marker should scale smaller into the background or larger into the foreground when the map is pitched and rotated
+ * @param {boolean=} [options.zoomScale=false] Whether the marker should scale with the map as you zoom in and out
+ * @param {number=} [options.nativeZoom=map.getZoom()] The native zoom level of the marker. If the map is at this zoom the marker will not be scaled. If zoom is greater the marker will be enlarged. If zoom is less the marker will be shrunk.
+ * @param {number=} [options.zoomFactor=1.6] The multiple by which to scale the size of the marker between each zoom level
```

Comment:
That's what I mean; that "little math" is exactly what the function is supposed to be computing ;P If that logic needs to be duplicated in the userland code it may as well be in Marker.js instead.

`zoom-interpolated` seems like it'd probably work; it'd probably need a `base` parameter too as a scale factor. I'd say `interpolated` is bad nomenclature because it specifically means to "insert (something) between fixed points".

Maybe `exponential-overflow` instead:

```
{
    type: 'exponential-overflow',
    stops: [[12, 1]],
    base: 2
}
```

It would work the exact same as `exponential` but when outside the bounds of the stops passed it would calculate an inferred overflow value.